tools/internal_ci/linux/grpc_xds_k8s_insecure.sh

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env bash
+# Copyright 2021 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex -o igncr || set -ex
+
+# Constants
+readonly GITHUB_REPOSITORY_NAME="grpc"
+# GKE Cluster
+readonly GKE_CLUSTER_NAME="interop-test-psm-sec-v2-us-central1-a"
```

Comment:
I thought we were considering a separate cluster for non-security tests. I'm OK with this as a temporary measure, but then we'd have to backport cluster change to all branches created from now on.
cc @lidizheng

Reply:
What could be a breaking change for clusters?

Reply:
It's not that we worry about a breaking change, it's that PSM Sec cluster is configured differently, and comes with an overhead for security-related services running along with kubernetes. Using a separate cluster for regular tests just seems simpler. @sanjaypujare what do you think?

Reply:
That make sense. I can migrate the url-map tests as well.

Reply:
That'll be great!

Reply:
Will create separate clusters for these tests